playlist.go

```diff
@@ -44,6 +45,19 @@ func (c *Client) Playlist(idOrUID string) (*Playlist, error) {
 	return playlist, nil
 }
 
+// Playlists fetches and returns a Grafana playlist.
+func (c *Client) Playlists(params url.Values) (*[]Playlist, error) {
+	path := "/api/playlists/"
+	playlists := &[]Playlist{}
+
+	err := c.request("GET", path, params, nil, playlists)
```

Comment:
Suggest not making a pointer to a list, and definitely not returning one.
```suggestion
// Playlists fetches and returns all Grafana playlists.
func (c *Client) Playlists(params url.Values) ([]Playlist, error) {
	path := "/api/playlists/"
	playlists := []Playlist{}

	err := c.request("GET", path, params, nil, &playlists)
```

Reply:
Sorry for the slow reply.

I have no strong feelings in the matter. 
I was trying to be consistent with the PlayList function
https://github.com/grafana/grafana-api-golang-client/blob/21beb0882a17695e1616e5f285b5eafcb1074fab/playlist.go#L36